cmd/cstor-volume-mgmt/controller/volume-controller/handler.go

```diff
@@ -118,8 +118,16 @@ func (c *CStorVolumeController) cStorVolumeEventHandler(
 		if err != nil {
 			return common.CVStatusError, err
 		}
+		//TODO: Should we need to make update call here itself or caller of the
+		//code will take care?
 
-		return common.CVStatusIgnore, nil
+		// IsZero returns true if the quantity is equal to zero.
+		if cStorVolumeGot.Status.Capacity.IsZero() {
+			// update the status capacity of cstorvolume caller of this code
+			// will update in etcd
+			cStorVolumeGot.Status.Capacity = cStorVolumeGot.Spec.Capacity
+		}
+		return common.CVStatusInit, nil
 
 	case common.QOpModify:
```

Comment:
in QOpModify as well, capacity need to be updated.. and need to be returned success

Reply:
I'm also seeing that - if any error happens here, CreateVolumeTarget won't get called again.. this can become the problem right?

Reply:
It will execute again since we are not removing object from queue when error happened